5-Heroku/server/routes/todos.js

```diff
@@ -0,0 +1,59 @@
+const express = require("express");
+const { v1: uuidv1, v4: uuidv4 } = require("uuid");
+const cookieParser = require("cookie-parser");
+const ToDoModel = require("../models.js");
+const {checkCookies} = require("../middleweres/cookies");
+const router = express.Router();
+
+router.use(cookieParser());
+router.use(express.json());
+router.use(checkCookies);
+
+
+router.get("/", async (req, res) => {
+  const _id = req.cookies.userId;
+  const toDoList = await ToDoModel.findById(_id).clone();
```

Comment:
what is the purpose of `clone`?

Reply:
The purpose of clone is to enable querying twice the same query, since it has been disabled in the latest version of mongoose.